licenses.yaml

```diff
@@ -2831,7 +2831,7 @@ libraries:
 notices:
   - zookeeper: |
       Apache ZooKeeper
-      Copyright 2009-2012 The Apache Software Foundation
```

Comment:
Oops, you've updated the notice for the zookeeper bundled in the hadoop-client, instead of the one in core druid that you actually updated the version for (notice starts on line 1458).

Reply:
👍